common/test/acceptance/pages/studio/index.py

```diff
@@ -205,12 +205,13 @@ def select_item_in_autocomplete_widget(self, item_text):
         )
         self.q(css='.ui-autocomplete .ui-menu-item a').filter(lambda el: el.text == item_text)[0].click()
 
-    def list_courses(self):
+    def list_courses(self, archived=False):
```

Comment:
Because of the default value, existing usages of this method will end up verifying (indirectly) that active courses are on the active list instead of the archived list.